### PR TITLE
Map block completed

### PR DESCRIPTION
**Pull request by dieselT**
@BggClr, @ilia280 can you review it, please?

### CODE REVIEW

File: Models/MapBlock.cs [hunk 1]
```diff
@@ -0,0 +1,11 @@
+using UmbracoGridConfigLoader.Models;
+using UmbracoGridConfigLoader.Attributes;
+
+namespace Graph.Components.MapBlock
+{
+	public class MapBlock : IGridConfigLoader
+	{
+		[GridLayoutProperty(Label = "Map Block")]
```
Comment (ilia280) [inline Models/MapBlock.cs]: @dieselT please add `MaxItems = 1` the GridLayoutProperty attribute

File: README.md [hunk 2]
```diff
@@ -7,4 +7,5 @@
 2. Install Npm packages:
 * npm i leaflet --save
 * npm i leaflet-gesture-handling --save
-3. Copy all files to the folder 'MapBlock' in ~\App_Plugins\
\ No newline at end of file
+3. Install [Content Blocks](https://github.com/graphuk/components_umbraco_content_blocks) (via Stamp)
```
Comment (ilia280) [inline README.md]: @dieselT please add stamp command to install `Content Blocks`

File: src/Models/MapBlock.cs [hunk 3]
```diff
@@ -5,7 +5,7 @@ namespace Graph.Components.MapBlock
 {
 	public class MapBlock : IGridConfigLoader
 	{
-		[GridLayoutProperty(Label = "Map Block")]
+		[GridLayoutProperty(Label = "Map Block", AllowedEditors = new []{"mapBlock"})]
```
Comment (ilia280) [inline src/Models/MapBlock.cs]: @dieselT, UmbracoGridConfigLoader package has been updated to 1.0.5 version
please add MaxItems=1

File: README.md (hunk 2 above)
Comment (dieselT) [reply README.md]: duplicated below

File: src/Models/MapBlock.cs (hunk 3 above)
Comment (dieselT) [reply src/Models/MapBlock.cs]: done

File: Models/MapBlock.cs (hunk 1 above)
Comment (dieselT) [reply Models/MapBlock.cs]: duplicated below

**Pull request merged**